doc/page-features.h

```diff
@@ -151,4 +151,22 @@ foo.userData<Pod>().i;  // 42
 An interface implementation can expose a structure containing some user data. This user data is
 preserved when the structure is copied. Thus it can be made available in a different interface.
 
+\section ServerImplementation   Implementation of Server inside C++ Implementation of Interface 
+
+In order to skip using IPC.enabled:true inside qml implementation , and create server inside C++ delegate, one
+can do the following 
```

Comment:
remove trailing whitespace

Reply:
I don't know what a "C++ delegate" is

Reply:
C++ delegate here i meant the delegate ( the one  who is implemented and then composed  to delegate the calls from qmlfrontend to the  underlying logic)